src/app.js

```diff
@@ -1,5 +1,25 @@
-import { layout } from './layout';
+import { layout, showScores } from './layout';
+import { createGame, addScore, getScores } from './api';
 import './style.css';
 
 const container = document.querySelector('.scores-container');
-container.innerHTML = layout();
\ No newline at end of file
+container.innerHTML = layout();
+const form = document.getElementById('form');
+const scoreList = document.getElementById('score-list');
+const refreshButton = document.getElementById('refreshBtn');
+
+window.addEventListener('DOMContentLoaded', async () => {
+  const gameId = (str) => createGame(str);
+
+  form.addEventListener('submit', async (event) => {
+    event.preventDefault();
+    const name = document.getElementById('formName');
+    const score = document.getElementById('formScore');
+    await addScore(name, score, gameId);
+  });
```

Comment:
## [OPTIONAL]
* I suggest you clear all input fields after adding a score this would improve user experience